### PR TITLE
VPN-5282 - test unauth feedback form

### PR DESCRIPTION
**Pull request by mozrokafor**
## Description

    automation test for VPN-5282 - unauth feedback form

## Reference

    https://mozilla-hub.atlassian.net/browse/VPN-5282

### CODE REVIEW

File: tests/functional/testContactUs.js [hunk 1]
```diff
@@ -32,6 +32,26 @@ describe('Contact us view', function() {
         queries.screenGetHelp.contactSupportView.UNAUTH_USER_INPUTS.visible());
   });
 
+  it('Verify unauth feedback filling out', async () => {
```
Comment (strseb) [inline tests/functional/testContactUs.js:35]: First of: I do know this is how other test's have been written in the past - so in that context that would be an r+. Just some thoughts: 

I'm having a hard time just looking at the test understanding what the test is doing, when it passes and when it should fail. It seems to fill out text fields? Or is this test about that it's writiable if the user is not authenticated? Can we make this a bit more decreptive, so when we come back later we know what this thing is testing and what it's not. 

The only failing conditions are:
-  `USER_INFO:visible` -> That assert does not have text, why should it not be visible, (i suppose it's probably becuase are supposed to not be logged in? ) can we add some text here so we understand why that may not be visible? :) 
- Global Timeout: If we hit that, no other tests will be run, so we won't know if only this one is failing or all in the suite as you need to fix them one by one.  Should we maybe give each unit (or like only this one for start) it's own timeout, so we do not abort the whole suite?

**Review by strseb — APPROVED** [review]
r+wc

File: src/apps/vpn/ui/screens/getHelp/contactUs/ViewContactUsForm.qml [hunk 2]
```diff
@@ -87,7 +87,7 @@ MZViewBase {
 
                     MZTextField {
                         id: emailInput
-
+                        objectName: "Email-Textfield"
```
Comment (MattLichtenstein) [inline src/apps/vpn/ui/screens/getHelp/contactUs/ViewContactUsForm.qml:90]: nit: our object names are usually written in camelcase eg `emailTextField`

**Review by MattLichtenstein — APPROVED** [review]

File: tests/functional/testContactUs.js (hunk 1 above)
Comment (mozrokafor) [reply tests/functional/testContactUs.js:35]: Will address this in a follow-up solution

**Pull request closed without merging**